24.md

```diff
@@ -0,0 +1,57 @@
+NIP-24
+======
+
+Protect the Unknown
+-------------------
+
+`draft` `optional` `client` `author:giszmo`
+
+## Replaceable Events
+
+A *replaceable event* like `kind: 0`, `kind: 3` and as defined in
+[NIP-16](16.md).
```

Comment:
maybe update to include nip 33 too?

A *replaceable event* like `kind: 0`, `kind: 3` and as defined in
[NIP-16](16.md) and [NIP-33](33.md).